README.md

```diff
@@ -125,3 +125,8 @@ A: Our build system requires Java 7 or later. If you see this error, install Jav
 	 You will also need to specify the new JDK version in Android Studio. Refer to this Stack Overflow entry for help doing so:
 
 	 http://stackoverflow.com/questions/30631286/how-to-specify-the-jdk-version-in-android-studio
+
+Q: After I upgraded to Android Studio v2.3, I've been facing alot of issues while compiling/building the project. How do I fix this?
```

Comment:
hmm nice points! I also felt like the FAQ items look abit stale. We should use markdown to improve readability.